### ISSUE TITLE
Cannot discover unit tests

### ISSUE THREAD

**Issue by richardschneider** [richardschneider/net-ipfs-core#83]
I'm using VS 2017 and with the latest pull  from `master` I can no longer run the unit tests.  VS cannot discover any tests because it cannot find `System,RunTime`.

```
[14/06/2019 2:16:38 PM Informational] ------ Discover test started ------
[14/06/2019 2:16:41 PM Warning] [MSTest][Discovery][C:\Users\Owner\Documents\GitHub\net-ipfs-core\test\bin\Debug\net45\IpfsCoreTests.dll] 
Failed to discover tests from assembly C:\Users\Owner\Documents\GitHub\net-ipfs-core\test\bin\Debug\net45\IpfsCoreTests.dll. 
Reason:Could not load file or assembly 'System.Runtime, Version=4.0.20.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
[14/06/2019 2:16:41 PM Warning] No test is available in C:\Users\Owner\Documents\GitHub\net-ipfs-core\src\bin\Debug\netstandard14\Ipfs.Core.dll C:\Users\Owner\Documents\GitHub\net-ipfs-core\test\bin\Debug\net45\IpfsCoreTests.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.
```

cc: @hanabi1224

**Comment by richardschneider** [comment]
I'm installing the latest version of VS 2017. Hopefully it will resolve it.

**Comment by richardschneider** [comment]
The CI is no longer able to build the documentation
```
[19-06-14 02:00:16.337]Warning:[MetadataCommand.ExtractMetadata]Fail to find MSBuild >= 16.0 on machine. Please install Visual Studio 2019 with MSBuild >= 16.0: https://visualstudio.microsoft.com/vs/
[19-06-14 02:00:18.201]Warning:[MetadataCommand.ExtractMetadata](C:/projects/net-ipfs-core/src/IpfsCore.csproj)Workspace failed with: [Failure] Msbuild failed when processing the file 'C:\projects\net-ipfs-core\src\IpfsCore.csproj' with message: The SDK 'Microsoft.NET.Sdk' specified could not be found.  C:\projects\net-ipfs-core\src\IpfsCore.csproj
[19-06-14 02:00:18.328]Warning:[MetadataCommand.ExtractMetadata]Project 'C:\projects\net-ipfs-core\src\IpfsCore.csproj' does not contain any documents.
````

@hanabi1224 It looks like your previous commit is forcing VS 2019 to be used!

**Comment by richardschneider** [comment]
@hanabi1224  If I revert you last two commits it is working.   I'll incrementally add your changes to hopefully find the issue.

**Comment by hanabi1224** [comment]
@richardschneider sry about that.  I dont have vs2017 right now, bit it must be Microsoft.NET.Test.Sdk  Try use 15.x it should fix the problem

**Comment by richardschneider** [comment]
It looks like two issues are present
- The latest docfx is requiring vs 2019
- SimpleBase v1.4.1 is causing the System.RunTime error

So, I'll go ahead and create a branch and see what happens.  Hopefully I will not revert your great changes.

**Comment by richardschneider** [comment]
Fixed via #85
